src/components/BookList.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable semi */
+import React from 'react';
+import AddBook from './BookInput';
+
+const BookList = () => (
+  <div className="book-list">
+    <AddBook />
+  </div>
+)
+
+export default BookList;
```

Comment:
- [ ] I think here you are missing the `Book` component, imported here, and the props which I talked about before should come from here, aka BookList => props => Book

Don't you think?